### PR TITLE
task-NXP-32000-Fork-Nuxeo-Explorer-2023

### PR DESCRIPTION
**Pull request by NourNuxeo**

### CODE REVIEW

File: ftests/explorer-ftests-base/src/main/java/org/nuxeo/functionaltests/explorer/pages/artifacts/ComponentArtifactPage.java [hunk 1]
```diff
@@ -91,18 +91,18 @@ public void checkAlternative() {
...
-                "Component org.nuxeo.ecm.core.api.versioning.VersioningService", "In bundle org.nuxeo.ecm.core",
-                "Documentation\n" + "Aliases\n" + "Resolution Order\n" + "Start Order\n" + "Implementation\n"
-                        + "Services\n" + "Extension Points\n" + "XML Source");
-        checkRequirements(null);
-        checkImplementationText("org.nuxeo.ecm.core.versioning.VersioningComponent");
+    public void checkSchedulerComponent() {
+        checkCommon("Component org.nuxeo.ecm.core.scheduler.SchedulerService",
+                "Component org.nuxeo.ecm.core.scheduler.SchedulerService", "In bundle org.nuxeo.ecm.core.event",
+                "Documentation\n" + "Requirements\n" + "Aliases\n" + "Resolution Order\n" + "Start Order\n" + "Implementation\n"
+                        + "Services\n" + "Extension Points\n" + "Contributions\n" + "XML Source");
```
Comment (NourNuxeo) [inline ftests/explorer-ftests-base/src/main/java/org/nuxeo/functionaltests/explorer/pages/artifacts/ComponentArtifactPage.java:98]: I did not add a check on Contributions value. There is a method which does it in the `OperationArtifactPage` (#checkContributingComponentText) only so I assumed this was not necessary. Maybe the list of contributions to a component "move" more frequently than contribs to XP points and we don't want to update the expected value frequently.

File: ci/Jenkinsfiles/build.groovy [hunk 2]
```diff
@@ -107,10 +107,17 @@ pipeline {
...
               Run Functional Tests
               ----------------------------------------"""
-              echo "MAVEN_OPTS=$MAVEN_OPTS"
-              sh "mvn ${MAVEN_ARGS} -f ftests verify"
-              nxUtils.lookupText(regexp: ".*ERROR.*(?=(?:\\n.*)*\\[.*FrameworkLoader\\] Nuxeo Platform is Trying to Shut Down)",
-                  fileSet: "ftests/**/log/server.log", unstableIfFound: true)
+              withCredentials([string(credentialsId: 'instance-clid', variable: 'INSTANCE_CLID')]) {
+              sh(script: '''#!/bin/bash +x
+                echo -e "$INSTANCE_CLID" >| /tmp/instance.clid
+              ''')
```
Comment (kevinleturc) [inline ci/Jenkinsfiles/build.groovy]:
```suggestion
                sh(script: '''#!/bin/bash +x
                  echo -e "$INSTANCE_CLID" >| /tmp/instance.clid
                ''')
```

**Review by kevinleturc — COMMENTED** [review]
Could the commit message be renamed to something like _Update the addon for Nuxeo LTS 2023_?

**Review by kevinleturc — COMMENTED** [review]
Looks good 👍.

Once the pipeline has finished, renamed your branch locally to `2023` and push it, then delete this branch and PR.

Once the 2023 pipeline will succeed, we could ask NCO to update the explorer.beta.nuxeocloud.com with Nuxeo LTS 2023 + the package which has been uploaded to Connect Preprod. If everything goes well we will be able to release the package for 2023 and start exporting Nuxeo LTS 2023.

**Pull request closed without merging**